### PR TITLE
Feature/email verification

### PR DESCRIPTION
**Pull request by tomyRootstrap**
## Links:

https://trello.com/c/fdVNd75T/17-3-as-a-user-i-should-be-able-to-confirm-my-account-after-signing-up-in-order-to-ensure-that-all-the-users-email-are-valid

## What & Why:

I've develop the integration with Emailjs in order to have an accurate way to validate the email that the user used on the registration flow.

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

**Review by FlaviaCohen — APPROVED** [review]
I insist about creating atomic commits. It's a good practice to make smaller commits that include the code of each small task with a descriptive name instead of one or two huge commits.

File: src/pages/Signup/index.js [hunk 1]
```diff
@@ -47,8 +48,30 @@ const Signup = () => {
...
 
-  const onSubmit = data => signup(data);
+  const [toSend, setToSend] = useState({
+    to_name: '',
+    message: `Hello, thank you for signing up to MVD! We're excited to have you onboard and will be happy to help you set everything up.`,
+    to: '',
+  });
+
+  const sendEmailValidation = data => {
+    setToSend((toSend.to_name = data.username), (toSend.to = data.email));
```
Comment (alandours) [inline src/pages/Signup/index.js]: You shouldn't mutate `toSend` directly, instead you should do it with `setToSend` passing a new object with the updated values. If you need to, you can access the previous state inside of `setToSend` (https://reactjs.org/docs/hooks-reference.html#functional-updates)

```suggestion
    setToSend((prevState) => {
       return {
          ...prevState,
          to_name: data.username,
          to: data.email
       }
    });
```

**Comment by tomyRootstrap** [comment]
> I insist about creating atomic commits. It's a good practice to make smaller commits that include the code of each small task with a descriptive name instead of one or two huge commits.

I forgot this practice, I will try to implement it in the future.

File: src/pages/Signup/index.js (hunk 1 above)
Comment (tomyRootstrap) [reply src/pages/Signup/index.js]: I will fix this now. Thanks.

**Review by alandours — APPROVED** [review]

**Pull request merged**